store/postgres/src/chain_store.rs

```diff
@@ -1177,18 +1424,10 @@ impl EthereumCallCache for ChainStore {
         } else {
             // No entry with the new id format, try the old one.
             let old_id = old_contract_call_id(&contract_address, &encoded_call, &block);
-            if let Some(return_value) = eth_call_cache::table
-                .find(old_id.as_ref())
-                .select(eth_call_cache::return_value)
-                .get_result::<Vec<u8>>(conn)
-                .optional()?
-            {
-                use public::eth_call_cache::dsl;
-
+            if let Some(return_value) = self.storage.get_call(conn, &old_id)? {
                 // Migrate to the new format by re-inserting the call and deleting the old entry.
                 self.set_call(contract_address, encoded_call, block, &return_value)?;
-                diesel::delete(eth_call_cache::table.filter(dsl::id.eq(old_id.as_ref())))
-                    .execute(conn)?;
+                self.storage.clear_cache(conn, old_id.as_ref())?;
```

Comment:
Now you've already gone through the trouble, but it is fine to drop this migration code by now.

Reply:
Just pushed one more commit to get rid of the migration code.

Reply:
Nice thanks